components/user-info/UserFavourites.tsx

```diff
@@ -2,24 +2,19 @@ import getItemDetails from '@/utils/fetchItemDetails'
 import ItemCard from '@/components/cards/ItemCard'
 import fetchSellerName from '@/utils/fetchSellerName'
 import getUser from '@/utils/getUser'
-import { Item } from '@/utils/types'
 
 export default async function Purchase() {
   const { supabase, userID } = await getUser()
 
-  let itemDetails: Item[] = []
-  if (userID) {
-    itemDetails = (await getItemDetails(
-      supabase,
-      'favourite_items',
-      userID,
-    )) as Item[]
-    itemDetails = itemDetails.filter(item => item !== undefined)
+  if (!userID) {
+    return null
```

Comment:
maybe this isn't the right thing to return, I was being lazy

I think it should be impossible for a logged out user to reach this component so this could actually be a legit time to use `userID!` on line 13!